src/components/SchedulePage/Event.js

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { makeStyles } from '@material-ui/core/styles';
+import Container from '@material-ui/core/Container';
+import Grid from '@material-ui/core/Grid';
+import Typography from '@material-ui/core/Typography';
+
+const useStyles = makeStyles(theme => ({
+	name: {
+		fontFamily: theme.typography.fontFamily,
+		color: theme.palette.secondary.main,
+		fontWeight: 'bold',
+		textTransform: 'uppercase'
+	},
+	location: {
+		fontFamily: theme.typography.fontFamily,
+		fontWeight: theme.typography.fontWeightMedium
+	},
+	description: {
+		fontFamily: theme.typography.fontFamily
```

Comment:
As a sidenote, I've noticed that most of our text is styled using Poppins in the figma, perhaps we can change the defaults in `layout.js` to Poppins and overriding in individual components when we want to use other fonts.

Reply:
It looks like we have to default formatting comments. Should I fix this here, or might it be better to do it in a new PR?

Reply:
> default formatting comments

what do you mean by that?

Reply:
Idk how I didn't catch this. What I wanted to say was "**two** default formatting comments suggesting to put something in `layout.js`".

Reply:
oh yeah, I think it's fine for us to do that in a separate PR since it'll involve cleaning up other unrelated files as well